Resizer/AdapterAwareTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Sonata\MediaBundle\Resizer;
+
+use Imagine\Image\ImagineInterface;
+
+trait AdapterAwareTrait
```

Comment:
For the reason that adapter and its method-accesors using in two classes at least